src/python/platforms/android/emulator.py

```diff
@@ -82,8 +77,7 @@ def run(self):
       raise EmulatorError('Attempted to `run` emulator before calling `create`')
 
     logs.log('Starting emulator.')
-    self.process = self.process_runner.run(
-        stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    self.process = self.process_runner.run(stdout=subprocess.PIPE)
```

Comment:
Since we don't care about stderr, let's set stderr to open(os.devnull, 'w').

Reply:
Done, but I'm using subprocess.DEVNULL.